.goreleaser.yml

```diff
@@ -95,7 +101,9 @@ dockers:
       - linux-amd64
     build_flag_templates:
       - --platform=linux/amd64
-    skip_push: true
+    extra_files:
+      - builder/files/genesis-mainnet-v1.json
+      - builder/files/genesis-testnet-v4.json
```

Comment:
If I want to run a devnet, I just need to add a new genesis file here and use that? How do I specify starting bor using the devnet genesis?

Reply:
The files are just there, you don't need to add devnet one here. The idea is that you mount a volume with your file and use that to init `bor` using you own genesys file.